#This file contains all the paramaters that will be used by a ROS node to request the entire control table (eeprom and ram) of a Dynamixel servo.

#The id specified here is to indicate which servo to get the control table from
uint8 id 